### PR TITLE
docs: Added api document for Appium-Python-Client

### PR DESCRIPTION
**Pull request by ki4070ma**
Added api document for `Appium-Python-Client` which is generated from docstring in `Appium-Python-Client`.

 `docs` folder is generated from `appium/python-client`, so review can be skipped.

### Github pages
I've set up github pages from settings, so the doc will be available on below page after this PR merged.

https://appium.github.io/python-client-sphinx/

### TODO
- [x] Copy files from ki4070ma/python-client-sphinx
- [x] Update doc files along to the latest lib
- [x] Update README, scripts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,8 @@
-python-client-sphinx
```
Comment (ki4070ma) [inline README.md:1]: Update README

File: github_pages_deploy.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```
Comment (ki4070ma) [inline github_pages_deploy.sh:1]: Delete this file

File: docs/_static/doctools.js [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+ * small helper function to urlencode strings
+ */
+jQuery.urlencode = encodeURIComponent;
+
+/**
+ * This function returns the parsed url parameters of the
+ * current request. Multiple values per key are supported,
+ * it will always return arrays of strings for the value parts.
+ */
+jQuery.getQueryParameters = function(s) {
```
Comment (mykola-mokhnach) [inline docs/_static/doctools.js:47]: heh, jQuery

oldschool

File: docs/_static/documentation_options.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+var DOCUMENTATION_OPTIONS = {
+    URL_ROOT: document.getElementById("documentation_options").getAttribute('data-url_root'),
+    VERSION: '1.0',
```
Comment (mykola-mokhnach) [inline docs/_static/documentation_options.js:3]: How do we set language and version?

Comment (ki4070ma) [reply docs/_static/documentation_options.js:3]: It's by `conf.py`.

https://github.com/appium/python-client/blob/master/docs/conf.py
   * version

https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-language
   * language

Comment (mykola-mokhnach) [reply docs/_static/documentation_options.js:3]: should we set the default language to English?

Comment (ki4070ma) [reply docs/_static/documentation_options.js:3]: I agree.
I'll update parameters in python-client repo and here.
(For this PR, I'll merge and check if doc is available, though.)

**Comment by KazuCocoa** [comment]
yey

**Pull request merged**